src/main/java/frc/utils/applicationsutils/OutputFile.java

```diff
@@ -0,0 +1,96 @@
+package frc.utils.applicationsutils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.awt.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class OutputFile {
+
+    private static final OutputFile SELF_OUTPUT_FILE = new OutputFile("OutputFile");
```

Comment:
```suggestion
    private static final OutputFile FILES_ERROR_OUTPUT_FILE = new OutputFile("FilesErros");
```
or something like it